src/bindings/cpp/examples/cpp_example_hierarchy.cpp

```diff
@@ -70,7 +70,7 @@ unsigned long name_depth (Key k)
 class Visitor
 {
 public:
-	virtual void visit (std::string name, unsigned long depth, Key k) = 0;
+	virtual void visit (std::string const & name, unsigned long depth, Key k) = 0;
```

Comment:
Yes they do support some c++11 things too, though maybe not this one. But as the article mentions it seems to depend on the use case anyway which variant is better, so i think sticking to the usual const reference should be fine in general.